.github/CODEOWNERS

```diff
@@ -43,3 +48,18 @@ Makefile                    @getsentry/owners-sentry-dev
 .envrc                      @getsentry/owners-sentry-dev
 .pre-commit-config.yaml     @getsentry/owners-sentry-dev
 .git-blame-ignore-revs      @getsentry/owners-sentry-dev
+
+
+
+# Owners by product feature
+/src/sentry/static/sentry/app/views/settings/projectAlerts/     @getsentry/workflow
+/src/sentry/static/sentry/app/views/settings/incidentRules/     @getsentry/workflow
+/src/sentry/static/sentry/app/views/alerts/                     @getsentry/workflow
+/tests/js/spec/views/settings/projectAlerts/                    @getsentry/workflow
+/tests/js/spec/views/settings/incidentRules/                    @getsentry/workflow
+/tests/js/spec/views/alerts/                                    @getsentry/workflow
+/src/sentry/incidents/                                          @getsentry/workflow
+/tests/sentry/incidents/                                        @getsentry/workflow
+/tests/snuba/incidents/                                         @getsentry/workflow
+/tests/acceptance/test_incidents.py                             @getsentry/workflow
+/src/sentry/templates/sentry/emails/incidents/                  @getsentry/workflow
```

Comment:
Could you also include these?: 
 - `src/sentry/snuba/models.py`
 - `src/sentry/snuba/query_subscription_consumer.py`
 - `src/sentry/snuba/subscriptions.py`
 - `tests/sentry/snuba/test_query_subscription_consumer.py`
 - `tests/sentry/snuba/test_subscriptions.py`
 - `tests/sentry/snuba/test_tasks.py`
 - `tests/snuba/snuba/test_query_subscription_consumer.py`